pkg/server/status.go

```diff
@@ -472,6 +472,7 @@ type statusServer struct {
 	si                       systemInfoOnce
 	stmtDiagnosticsRequester StmtDiagnosticsRequester
 	internalExecutor         *sql.InternalExecutor
+	tenantStatusServer       serverpb.TenantStatusServer
```

Comment:
`t.sqlServer.tenantConnect` already contains `tenantStatusServer` so we can just use that instead.

Reply:
See `statusServer.TenantRanges` endpoint for reference.

Reply:
Done. Thanks for the tip, @shubhamdhama!